openroad.bzl

```diff
@@ -719,7 +720,7 @@ def build_openroad(
         stage_sources["floorplan"].append(io_constraints)
         stage_sources["place"].append(io_constraints)
     stage_sources["route"] += outs["cts"]
-    for stage in ["floorplan", "final", "generate_abstract"]:
+    for stage in ["synth", "floorplan", "final", "generate_abstract"]:
```

Comment:
I don't think LEF files are needed during synth stage, but they should not break anything

Reply:
@hovind @eszpotanski LEF files are used during synthesis:

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/c1a0f94aa3950b1506ad7da2e5bd3c6df7d40f52/flow/scripts/load.tcl#L11C1-L21C1